server/index.js

```diff
@@ -31,6 +33,12 @@ module.exports = function(options) {
     }
   });
 
+  server.state("stripeCustomerId", {
+    ttl: ONE_HOUR_MS,
+    isSecure: process.env.NODE_ENV === "production",
+    encoding: "base64"
```

Comment:
Should we use something like [Iron](https://www.npmjs.com/package/iron) to encrypt/decrypt this server-side?

I'd prefer the client to not even be aware of what the customer ID even is, and for it to be difficult to craft a fake cookie.

Reply:
sure, I can add that